### ISSUE TITLE
Obsolete values in TEST default configuration

### ISSUE THREAD

**Issue by thodt-md** [open-eid/digidoc4j#36]
Since https://demo.sk.ee/TSL/tl-mp-test-EE.xml is not available anymore, the updated URL from https://github.com/open-eid/digidoc4j/wiki/Using-test-TSL-lists should also be present in `org.digidoc4j.Configuration`.

If I try refreshing TSL with the default TEST configuration, it fails:
```
...
Caused by: org.digidoc4j.exceptions.TslCertificateSourceInitializationException: Unable to load the LOTL: content is empty
	at org.digidoc4j.impl.bdoc.tsl.LazyTslCertificateSource.refreshTsl(LazyTslCertificateSource.java:139)
	at org.digidoc4j.impl.bdoc.tsl.LazyTslCertificateSource.refresh(LazyTslCertificateSource.java:93)
...
```

**Comment by thodt-md** [comment]
BTW any plans to refactor (or better yet, rewrite) this class? Right now it's responsible for storing the default values for two environments, YAML parsing/binding, validation, configuration logging and whatnot. Most of that can be done [somewhere](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/context/properties/ConfigurationProperties.html) [else](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/validation/annotation/Validated.html).

**Comment by nostneji** [comment]
Described problem is solved in release 1.0.7. And we have plans to refactor this code ... probably in next release.
